### ISSUE TITLE
Cannot download videos while authenticated (`401 Unauthorized`)

### ISSUE THREAD

**Issue by th10010** [Tyrrrz/YoutubeDownloader#546]
### Version

v1.13.3

### Platform

windows11

### Steps to reproduce

When I use the software to download videos, a pop-up window similar to the picture will pop up when downloading videos. No matter whether I log out of the account or try other operations, the video cannot be downloaded successfully.
https://www.youtube.com/watch?v=wuxmD5H67sc
https://www.youtube.com/watch?v=BVHc1IUiT80

### Details

![1](https://github.com/user-attachments/assets/2aa13337-c242-4900-9869-fdadd1f4b82e)
![2](https://github.com/user-attachments/assets/4df4d781-1e2c-4f11-8cbf-90f5feeaedb2)

### Checklist

- [X] I have looked through existing issues to make sure that this bug has not been reported before
- [X] I have provided a descriptive title for this issue
- [X] I have made sure that this bug is reproducible on the latest version of the application
- [X] I have provided all the information needed to reproduce this bug as efficiently as possible
- [ ] I have sponsored this project
- [ ] I have not read any of the above and just checked all the boxes to submit the issue

**Comment by mkinit** [comment]
Log out and you can work again (from Google Translate)

**Comment by th10010** [comment]
> Log out and you can work again (from Google Translate)

I have tried logging in again, but still have this problem

**Comment by mkinit** [comment]
> > Log out and you can work again (from Google Translate)注销，您可以再次工作（来自 Google 翻译）
> 
> I have tried logging in again, but still have this problem我已尝试再次登录，但仍然遇到此问题

No need to log in. I don't know what this login function does. I can download it even when I'm not logged in. (Google Translate)

**Comment by mohamadpk** [comment]
same  as same i have this problem login or logout .you can just add a cookie reader from firefox or chrome or opera or ... like other youtube downloader

**Comment by 0xEutxo** [comment]
> ### Version
> v1.13.3
> 
> ### Platform
> windows11
> 
> ### Steps to reproduce
> When I use the software to download videos, a pop-up window similar to the picture will pop up when downloading videos. No matter whether I log out of the account or try other operations, the video cannot be downloaded successfully. https://www.youtube.com/watch?v=wuxmD5H67sc https://www.youtube.com/watch?v=BVHc1IUiT80
> 
> ### Details
> ![1](https://private-user-images.githubusercontent.com/55772277/385192609-2aa13337-c242-4900-9869-fdadd1f4b82e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE5NjA4NzYsIm5iZiI6MTczMTk2MDU3NiwicGF0aCI6Ii81NTc3MjI3Ny8zODUxOTI2MDktMmFhMTMzMzctYzI0Mi00OTAwLTk4NjktZmRhZGQxZjRiODJlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTE4VDIwMDkzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI1OTE1NmUyODZiYmNlNWY5NDEzNDFkODI5MzBmMGQwYTMxMDJjMGVkNDk4ZTA5YzAyNjVhMjgxZGVjODllNzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.CdvqrWR2e8r8jK1qttw6YJjFf5sraxcnBXPJ1bxVqPo) ![2](https://private-user-images.githubusercontent.com/55772277/385192651-4df4d781-1e2c-4f11-8cbf-90f5feeaedb2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE5NjA4NzYsIm5iZiI6MTczMTk2MDU3NiwicGF0aCI6Ii81NTc3MjI3Ny8zODUxOTI2NTEtNGRmNGQ3ODEtMWUyYy00ZjExLThjYmYtOTBmNWZlZWFlZGIyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTE4VDIwMDkzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRmNTBkMTA0Mjc5MjJiYjMyNGNiNGZjNGJiYThjY2FhZGI1MDM3OTcxYmUwODM1MzhmMTRhYWQzNmM5N2YwNmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.6FO1fqx5uGPqbLf1qEiQEyC2LE9kb08_FrKlhXNd0Gk)
> 
> ### Checklist
> * [x]  I have looked through existing issues to make sure that this bug has not been reported before
> * [x]  I have provided a descriptive title for this issue
> * [x]  I have made sure that this bug is reproducible on the latest version of the application
> * [x]  I have provided all the information needed to reproduce this bug as efficiently as possible
> * [ ]  I have sponsored this project
> * [ ]  I have not read any of the above and just checked all the boxes to submit the issue

Im having the exact same issue. logged in or out.

**Comment by 0xEutxo** [comment]
Im having the exact same issue. logged in or out.

**Comment by Tyrrrz** [comment]
Just to be clear, which issue are you having? The original post includes screenshots for two separate errors

**Comment by mohamadpk** [comment]
I said that it is exactly the same error with the same form and the same windows and the same error messages

**Comment by gautam-kumar71** [comment]
yeah the problem still persists , the same error keeps coming as i press the download button

**Comment by mohamadpk** [comment]
the problem is in YoutubeExplode ,i think

**Comment by xiaolingtong** [comment]
> Log out and you can work again (from Google Translate)

yes，It works

**Comment by Tyrrrz** [comment]
@th10010 just to confirm, which of the two errors are you getting when you're _not_ logged in the app?

**Comment by Tyrrrz** [comment]
Underlying issue: https://github.com/Tyrrrz/YoutubeExplode/issues/781

**Comment by JamaicanRumCream** [comment]
When I logged into my account, I am still unable to download a video. My account is non-premium, ie free. Using version 1.14.1 on Win 11.

![Image](https://github.com/user-attachments/assets/7f396a1f-b29d-4900-a24a-fe5d6f6ad4c8)
